modules/fileout_netcdf/FONcTransform.h

```diff
@@ -59,19 +59,20 @@ class BESDataHandlerInterface;
  */
 class FONcTransform: public BESObj {
 private:
-	int _ncid = {0};
-	libdap::DDS *_dds = {nullptr};
+    int _ncid = {0};
```

Comment:
Another FYI, when fields like ints and pointers are initialized, there is no need to use {}. The curly braces tell the compiler to use the object's constructor, but for those types, there is no constructor so it will just assign the value. Using the {} notation can make a big difference with fields that are objects.

Reply:
Yes, actually this is from the old code. I correct them.  However, I find quite a few private members not follow the _d naming conventions. I decide not to handle them. Instead I add a comment on ticket hyrax-1422 once I find time to clean up the fileout netCDF code.